### ISSUE TITLE
Android project build (in AzureDevOps CI, WindowsOS agent) started failing

### ISSUE THREAD

**Issue by knocte** [nblockchain/ZXing.Net.Xamarin#2]
Build log here:

https://dev.azure.com/diginex/ZXing.Net.Xamarin/_build/results?buildId=380&view=logs

Copy+pasting the error:

```
"D:\a\1\s\ZXing.Net.Mobile.sln" (Build target) (1) ->
"D:\a\1\s\Source\ZXing.Net.Mobile.Android\ZXing.Net.Mobile.Android.csproj" (default target) (5) ->
(_ValidateAndroidPackageProperties target) -> 
  C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\MSBuild\Xamarin\Android\Xamarin.Android.Common.targets(647,2): error XA5207: Could not find android.jar for API Level 15. This means the Android SDK platform for API Level 15 is not installed. Either install it in the Android SDK Manager (Tools > Open Android SDK Manager...), or change your Xamarin.Android project to target an API version that is installed. (C:\Program Files (x86)\Android\android-sdk\platforms\android-15\android.jar missing.) [D:\a\1\s\Source\ZXing.Net.Mobile.Android\ZXing.Net.Mobile.Android.csproj]

    244 Warning(s)
    1 Error(s)

Time Elapsed 00:00:39.71
An error occurred when executing task 'libs'.
```

**Comment by knocte** [comment]
@AndreiMisiukevich any idea? can you help here by any chance?

**Comment by AndreiMisiukevich** [comment]
@knocte seems, that our project has minimum sdk version 15 (?), but we didn't install proper version of sdk tools.

Can we manage it?

![image](https://user-images.githubusercontent.com/10124814/53326191-00521380-38f6-11e9-8df1-4301446263ad.png)

Not sure, just idea

**Comment by knocte** [comment]
But this is for CI, there's no GUI there.

But wait, hopefully I've fixed it with this: https://github.com/nblockchain/ZXing.Net.Xamarin/commit/8daadee20014f95b0c011a4fc54916431bf6ac4a

**Comment by AndreiMisiukevich** [comment]
@knocte looks good.
Hope it is what we need :)

**Comment by knocte** [comment]
Fixed in 50cb2621ae04cd121e0d260a2c66c6f564ac9500
